pkg/pillar/types/domainmgrtypes.go

```diff
@@ -474,6 +481,7 @@ type DomainMetric struct {
 	UsedMemoryPercent float64
 	LastHeard         time.Time
 	Activated         bool
+	NodeName          string // kube app running on node
```

Comment:
Maybe more clear comment would be: "the name of the kubernetes node on which the app is running"

Reply:
Sure.